packages/indexer-agent/src/agent.ts

```diff
@@ -696,6 +727,7 @@ class Agent {
     )
   }
 
+  // i don't think this needs to be changed?
```

Comment:
👍  Since you updated `deploymentsWorthAllocatingTowards()` to always select subgraphs with `decisionBasis.OFFCHAIN`.